diff --git a/test/test_helper.rb b/test/test_helper.rb
index 4e702e1..0a40b5b 100644
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -17,6 +17,11 @@ class ActiveSupport::TestCase
   # Add more helper methods to be used by all tests here...
 end
 
+OmniAuth.config.test_mode = true
+  @oauth_hash = OmniAuth::AuthHash.new({
+                                         provider: 'twitter', uid: '12345', info: { nickname: 'すいている', image: 'http://example.com/image.jpg' , point: 0 }
+                                       })
+  OmniAuth.config.mock_auth[:twitter] = @oauth_hash
 def login(h = {  })
   OmniAuth.config.test_mode = true
   OmniAuth.config.mock_auth[:twitter] = nil
